firedrake/preconditioners/pmg.py

```diff
@@ -46,7 +42,8 @@ class PMGBase(PCSNESBase):
     The default coarsen_element is to perform power-of-2 reduction
     of the polynomial degree.
     It is expected that some applications of this preconditioner
-    will subclass :class:`PMGBase` to override `coarsen_element`.
+    will subclass :class:`PMGBase` to override `coarsen_element` and
+    `coarsen_form`.
```

Comment:
This PC comes with lots of petsc options to twiddle. Can they be added to the docstring as well?